openpaas-james/mailbox/encrypted/memory/src/main/java/com/linagora/openpaas/encrypted/InMemoryKeyId.java

```diff
@@ -0,0 +1,14 @@
+package com.linagora.openpaas.encrypted;
+
+import java.util.UUID;
+
+public class InMemoryKeyId extends KeyId {
```

Comment:
I am in favor of having only one keyId object, even if it means we can't be using timeUuid in cassandra implem side. We do not need time ordering...

This step will make the code MUCH simpler